vizro-core/src/vizro/static/css/collapse.css

```diff
@@ -1,37 +0,0 @@
-@media (prefers-reduced-motion: reduce) {
```

Comment:
I think we've added this back then, because there was no bootstrap theme loaded by default, right? @nadijagraca  All of this shouldn't be required anymore given that there is a bootstrap CSS file loaded now. I checked the example apps and it seems to work fine 👍

Reply:
Just to let you know that there is no longer a smooth collapsing transition when the control panel is collapsed/expanded.

Reply:
Yeah, I noticed, but I didn't think it was that bad for it to need custom static CSS. I rather live with the default collapse behaviour from Bootstrap and have no static CSS 😬

Reply:
@maxschulz-COL @nadijagraca Do you have any thoughts on this? I think we should keep the collapse transition as done in the main branch, bug I'm okay to remove it if you agree.